token/cli/src/main.rs

```diff
@@ -2059,6 +2060,43 @@ async fn command_sync_native(config: &Config<'_>, native_account_address: Pubkey
     })
 }
 
+async fn command_withdraw_excess_lamports(
+    config: &Config<'_>,
+    source_account: Pubkey,
+    destination_account: Pubkey,
+    authority: Pubkey,
+    bulk_signers: Vec<Arc<dyn Signer>>,
+) -> CommandResult {
+    let token = native_token_client_from_config(config)?;
```

Comment:
Rather than using the native client, which can be confusing, let's explicitly use `Pubkey::default()` for the mint address, ie:
```suggestion
    // default is safe here because withdraw_excess_lamports doesn't use it
    let token = token_client_from_config(config, &Pubkey::default(), None)?;
```